src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -307,6 +307,40 @@ private static (string severityLabel, NuGetLogCode code) GetSeverityLabelAndCode
                 }
             }
             return result;
+
+            IEnumerable<(LibraryIdentity package, bool isDirect)> WalkAcceptedPackages(RestoreTargetGraph graph)
```

Comment:
We'll probably need a test for this method to make it easier to argue for a patch.